Zend/zend_compile.c

```diff
@@ -4198,6 +4187,35 @@ ZEND_API void zend_set_function_arg_flags(zend_function *func) /* {{{ */
 }
 /* }}} */
 
+
+static void _compile_typename(zend_ast *ast, zend_arg_info *arg_info) /* {{{ */
```

Comment:
I suppose I could name it `_compile_param_or_return_type` if you think that's better.